pgcontents/tests/test_hybrid_manager.py

```diff
@@ -88,6 +90,21 @@ def setUp(self):
     def test_get_file_id(self):
         pass
 
+    # This test also uses `get_file_id`, but it is not pertinent to the crucial
+    # parts of the test so just mock out.
+    def test_rename_file(self):
+        HybridContentsManager.get_file_id = Mock()
+        try:
+            super(PostgresTestCase, self).test_rename_file()
+        finally:
+            del HybridContentsManager.get_file_id
```

Comment:
This setup isn't great but I didn't have any other ideas at the moment other than doing this mocking, or removing this check: https://github.com/quantopian/pgcontents/pull/64/files#diff-ecde3e655b312f914854e679ceacaeccR195

Reply:
Ok